cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/BrokerSetFileResolver.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2022 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.config;
+
+import com.linkedin.kafka.cruisecontrol.exception.BrokerSetResolutionException;
+import com.linkedin.kafka.cruisecontrol.model.ClusterModel;
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.google.gson.Gson;
+import com.linkedin.kafka.cruisecontrol.KafkaCruiseControlUtils;
+
+
+/**
+ * BrokerSet information store implementation based out of File config broker.set.config.file
+ * By default the property is set to value of brokerSets.json
+ *
+ * Example Broker Set Data File :
+ * <pre>
+ * {
+ *   "brokerSets":[
+ *     {
+ *       "brokerSetId": "Blue",
+ *       "brokerIds": [0, 1, 2]
+ *     },
+ *     {
+ *       "brokerSetId": "Green",
+ *       "brokerIds": [3, 4, 5]
+ *     }
+ *   ]
+ * }
+ * </pre>
+ */
+public class BrokerSetFileResolver implements BrokerSetResolver {
+  private static final Logger LOG = LoggerFactory.getLogger(BrokerSetFileResolver.class);
+  public static final String BROKER_SET_CONFIG_FILE = "broker.set.config.file";
```

Comment:
We already have this string in analyzer config. We should probably remove this one?